src/simulator/views/map/display/solid_colour_map_display_pot.py

```diff
@@ -26,12 +27,11 @@ def __init__(self, services: Services, points, z_index=50, radius=0, custom_map:
         self.small = min(pmaplst)
         self.range = (self.large-self.small)/6
         self.pts = set()
+        self.__point_dim = None
 
-        #assert self._map.size.n_dim == 2, "Unsupported map dimension, expected 2D"
-
-    def render(self, refresh: bool) -> None:
+    def render(self, *discarded) -> None:
         points: Union[Set[Point], List[Entity]] = copy.deepcopy(self.points)
-        if points is None:
+        if not points:
```

Comment:
can we use `len(points) == 0`?

Reply:
Yh won't check for None anymore since this is not a case we want to be valid.

Reply:
Done.